server/api/api/endpoints/model_endpoints.py

```diff
@@ -389,21 +403,23 @@ async def _get_metrics_values_data(
     ],
     start: Optional[datetime] = None,
     end: Optional[datetime] = None,
+    aggregation_window: Optional[str] = None,
```

Comment:
Don't expose this in the API, as
1. We don't implement it for other metrics/results.
2. It's not a part of the agreed API.

Reply:
Alright.